src/Resources/translations/VerifyEmailBundle.ca.xlf

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" version="1.2">
+    <file original="file.ext" source-language="en" datatype="plaintext">
```

Comment:
Same here, let's add `target-language` attribute

Reply:
I've added the target language attribute, nevertheless a lot of other language files hasn't this attribute.

Reply:
May be not "a lot" but "some" ;-)

Reply:
Yeah, I see it now... I think we should fix those, and probably we could add this attr to en translation as well 🤔   - it would reduce such problems in the future, most devs just copy/paste the en template :)